test/test_values.py

```diff
@@ -93,3 +93,21 @@ def test_getting_values():
         "comment_parent": 0,
         "user_id": 0,
     }
+
+    parser = Parser(
+        "UPDATE `page_vote`"
+        "SET article_id = '442002', user_time = '27574631'"
+        "WHERE topic_id = '10000' AND section_id = '200000'"
+    )
```

Comment:
Can we cover here passing numeric values as well? Would be nice to cover the `_format_value` funcion that you introduced.

Reply:
@macbre Sure. Will do that.